### PR TITLE
feat: add create ROMP endpoint

### PR DESCRIPTION
**Pull request by johnytiago**

### CODE REVIEW

File: apps/users-service/src/cms/romps.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { Base, BaseOptions } from '@asap-hub/services-common';
+import { Romp } from '../entities/romps';
+
+export interface CreateRomp {
```
Comment (jeysal) [inline apps/users-service/src/cms/romps.ts:4]: We can throw this into `packages/model` already so it's accessible to FE+BE :upside_down_face:

File: apps/users-service/src/cms/index.ts [hunk 2]
```diff
@@ -1,11 +1,15 @@
 import Users from './users';
+import Romps from './romps';
```
Comment (jeysal) [inline apps/users-service/src/cms/index.ts:2]: The team's outputs overall are called the ROMP, so I think for the individual 'posts' we should find a different terminology. Maybe `ResearchOutput(s)` or just `Output(s)`? @fampinheiro

File: apps/users-service/test/handlers/users.create-romp.test.ts [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+import { config as authConfig } from '@asap-hub/auth';
+
+jest.mock('@asap-hub/auth');
+
+const chance = new Chance();
+
+describe('GET /users/{id}/romp', () => {
+  let id, code;
+
+  const romp = {
```
Comment (jeysal) [inline apps/users-service/test/handlers/users.create-romp.test.ts:16]: Should be typed as `const romp: CreateRomp = ...` to make sure properties are updated whenever we change the interface

File: apps/users-service/src/cms/index.ts (hunk 2 above)
Comment (fampinheiro) [reply apps/users-service/src/cms/index.ts:2]: I've been referring to them as `RompItems`, ReseachOutputs` is an option. Let me raise this to see the terminology product and design is using.

File: apps/users-service/test/handlers/users.create-research-output.test.ts [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+    code = user.connections[0].code;
+  });
+
+  test('return 401 when Authentication header is not set', async () => {
+    const result = (await handler(
+      apiGatewayEvent({
+        httpMethod: 'post',
+        body: output,
+        pathParameters: {
+          id,
```
Comment (fampinheiro) [inline apps/users-service/test/handlers/users.create-research-output.test.ts:50]: I'm wondering if we need the `id` in the header. We can extract it from the authenticated user.

**Pull request merged**